project/templates/drupal.py

```diff
@@ -1,15 +1,16 @@
-from . import BaseProject
-from .remote import RemoteProject
 import json
 from collections import OrderedDict
 
+from project.baseproject import BaseProject
+from project.remoteproject import RemoteProject
 
-class Drupal7_vanilla(BaseProject):
+
+class Drupal7Vanilla(BaseProject):
```

Comment:
Why this rename?  The template repository has an underscore in it, so I fear this would break it.

Reply:
PEP naming convention only (CamelCase class names). 

I’ll try running a build ...

Reply:
Seems to work fine, but fails on a patch so hard to say conclusively.

Reply:
I think this is one PEP 8 convention we can and _should_ violate. I've reverted this change (although it inexplicably doesn't seem to be showing up in the changes I just pushed).